MiniApp/Classes/core/MiniAppAnalytics.swift

```diff
@@ -10,13 +10,16 @@ internal enum MiniAppRATEvent: String, CaseIterable {
     case hostLaunch = "host_launch"
     case open
     case close
+    case signatureFailure = "signature_failure"
 
     func name() -> String {
-        return "mini_app_\(rawValue)"
+        "mini_app_\(rawValue)"
     }
 
     func eType() -> MiniAppRATEventType {
         switch self {
+        case .signatureFailure:
+            return .custom
```

Comment:
I don't think this is correct - we can only use pre-defined values for the `etype`. There aren't really any that make sense for this case, but maybe `async` is the closest? And then we would need to set another parameter to distinguish that this is the signature failure event. How about this?

```
etype = async
actype = signature_failure
```

Reply:
I just noticed that the following is what @khairul-alam-licon  used on Android, so probably this is okay:
```
etype = click
actype = mini_app_signature_validation_fail
```